config/templates/lambda.tpl.js

```diff
@@ -1,4 +1,4 @@
-export function handler(event, context) {
+export function handler(event, context, callback) {
     console.log(event);
-    context.success('done!');
-}
\ No newline at end of file
+    callback(null, 'done!');
```

Comment:
They're still two major versions behind though!  is 4.3 really the latest you can use?